### ISSUE TITLE
JAWS does not announce contenteditable div content on focus

### ISSUE THREAD

**Issue by BryanValverdeU** [FreedomScientific/VFO-standards-support#610]
### Summary

> JAWS is not announcing a div contenteditable content on focus
> 
> Example:
> 1. Go to https://codepen.io/bryanvalverdeu-the-typescripter/pen/popvVMj
> 2. Tab / Move to the content editable div element with text
> 3. Notice how only the aria label is announced but the text inside of div element is not announced

### Expected result

> On focus the div element, the content of the contenteditable div should be announced.
> If I move with down arrow, content is announced, but would be great if we could announce all the div content of focus

### Actual result

> Only the aria label is announced

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number

Version: 2022.2110.36 ILM

 
#### Operating System and version

Windows 11

#### Browser and version:

Able to repro in Edge 99, Chrome 99 and Firefox

**Comment by JAWS-test** [comment]
This is intentional and not a bug. This behaves analogously to the `<textarea>` element. Because the value can be very long, JAWS only indicates if a value is present, but does not output the value. This makes sense. If no aria-multiline is used, the value is also output when focusing.

**Comment by BryanValverdeU** [comment]
Thank you for the quick response, closing this as it is expected behavior.
